src/move_basic.cpp

```diff
@@ -367,6 +372,16 @@ void MoveBasic::executeAction(const move_base_msgs::MoveBaseGoalConstPtr& msg)
 
     pathPub.publish(path);
 
+    tf2::Transform goalInBase;
+    if (!transformPose(frameId, "base_link", goal, goalInBase)) {
+         ROS_WARN("Cannot determine robot pose for linear");
+         return;
+    }
+    tf2::Vector3 linear = goalInBase.getOrigin();
+    ROS_INFO("Forward = %f", linear.x());
+    bool straightBack = (linear.x() > -straightBackThreshold && linear.x() < 0.0);
```

Comment:
same with the variable name.

Also it may be more readable if you rearrange the expression to be like `-threshold < X && X < 0`.

Reply:
Changed